joss-paper/paper.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'PyFstat: a Python package for continuous gravitational-wave data analysis'
+tags:
+  - Python
+  - gravitational waves
+  - continuous waves
+  - pulsars
+  - data analysis
+authors:
+  - name: David Keitel
+    orcid: 0000-0002-2824-626X
+    affiliation: "1"
+  - name: Rodrigo Tenorio
+    orcid: 0000-0002-3582-2587
+    affiliation: "1"
+  - name: Gregory Ashton
+    orcid: 0000-0001-7288-2231
+    affiliation: "2"
+  - name: Reinhard Prix
+    orcid: 0000-0002-3789-6424
+    affiliation: "3, 4"
+affiliations:
+  - name: Departament de Física, Institut d'Aplicacions Computacionals i de Codi Comunitari (IAC3), Universitat de les Illes Balears, and Institut d'Estudis Espacials de Catalunya (IEEC), Crta. Valldemossa km 7.5, E-07122 Palma, Spain
+    index: 1
+  - name: OzGrav, School of Physics & Astronomy, Monash University, Clayton 3800, Victoria, Australia
+    index: 2
+  - name: Max-Planck-Institut für Gravitationsphysik (Albert-Einstein-Institut), D-30167 Hannover, Germany
+    index: 3
+  - name: Leibniz Universität Hannover, D-30167 Hannover, Germany
+    index: 4
+date: 08 January 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Gravitational waves in the sensitivity band of ground-based detectors
+can be emitted by a number of astrophysical sources,
+including not only binary coalescences, but also individual spinning neutron stars.
+The most promising signals from such sources,
+although as of 2020 not yet detected,
+are the long-lasting, quasi-monochromatic 'Continuous Waves' (CWs).
+Many search methods have been developed and applied on
+LIGO [@TheLIGOScientific:2014jea]
+and Virgo [@TheVirgo:2014hva] data.
+See @Prix:2009oha, @Riles:2017evm, and @Sieniawska:2019hmd for reviews of the field.
+
+The `PyFstat` package provides a convenient interface
+to perform a range of CW data analysis tasks.
+It revolves around the $\mathcal{F}$-statistic,
+first introduced by @Jaranowski:1998qm:
+a matched-filter detection statistic for CW signals
+described by a set of frequency evolution parameters
+and maximized over amplitude parameters.
+It has been one of the standard methods for LIGO-Virgo CW searches for two decades.
```

Comment:
Add reference to a search paper? O2 all-sky search?
@article{Pisarski:2019vxw,
    author = "Abbott, B. P. and others",
    collaboration = "LIGO Scientific, Virgo",
    title = "{All-sky search for continuous gravitational waves from isolated neutron stars using Advanced LIGO O2 data}",
    eprint = "1903.01901",
    archivePrefix = "arXiv",
    primaryClass = "astro-ph.HE",
    reportNumber = "LIGO-P1900012-v7",
    doi = "10.1103/PhysRevD.100.024004",
    journal = "Phys. Rev. D",
    volume = "100",
    number = "2",
    pages = "024004",
    year = "2019"
}

Reply:
Definitely not that one, as it uses the independent time-domain implementation. (Still F-stat, but confusing to point readers to.) E@H papers would be the most recent "close family members", but I'd say the Riles review reference earlier is sufficient?